cmd/eventkitd-sink-bigquery/go.mod

```diff
@@ -0,0 +1,38 @@
+module github.com/jtolio/eventkit/cmd/eventkitd-sink-bigquery
+
+go 1.18
+
+replace github.com/jtolio/eventkit => ../../
+
+require (
+	cloud.google.com/go/bigquery v1.42.0
+	github.com/jtolio/eventkit v0.0.0-20220927171925-c181046ab8ab
```

Comment:
TIL: You can run `go get -u github.com/jtolio/eventkit` and `go mod tidy` to update this version based on the replace.